tests/api_object.js

```diff
@@ -10,7 +10,6 @@ tape.test("reflection objects", function(test) {
     var obj = new protobuf.ReflectionObject("Test");
 
     obj.resolve();
-    test.equal(obj.resolved, false, "should not resolve when not part of a root");
```

Comment:
removing this test as we decided that `resolved` should be called on every level after it's been resolved, this will make future resolutions more efficient.